fs/log/redirect_stderr_windows.go

```diff
@@ -34,6 +35,6 @@ func setStdHandle(stdhandle int32, handle syscall.Handle) error {
 func redirectStderr(f *os.File) {
 	err := setStdHandle(syscall.STD_ERROR_HANDLE, syscall.Handle(f.Fd()))
 	if err != nil {
-		log.Fatalf("Failed to redirect stderr to file: %v", err)
+		fs.Fatalf(nil, "Failed to redirect stderr to file: %v", err)
```

Comment:
@ncw Is it correct to do it here?

Reply:
Hmm, not sure. Does it cause a stack overflow due to unbounded recursion? If not then the change is good. You might need to simulate a failure to test.

Reply:
It seems to work fine. Commenting out the if condition, running with `--log-file` option, the message "Failed to redirect stderr to file" ends up in the file. No stack overflow.